duckduckgo/iOS
# New image added for remote messages

Task/Issue URL: https://app.asana.com/0/0/1204559401135163/f
Tech Design URL:
CC:

**Description**:
Added new MacComputer image



**Steps to test this PR**:
1. Modify `RemoteMessageRequest` debug endpoint to this: https://jsonblob.com/api/1104418753722204160
2. Launch the app and open a new tab if not already on one; you should see the remote message
3. Confirm the new mac computer image is shown



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
